resources/views/welcome.blade.php

```diff
@@ -39,21 +39,20 @@
                     <div class="card-body">
                         @if(!$signedAnyAgreement)
                             <h4 class="card-title">
-                                Sign Membership Agreement
+                                Start Here
                             </h4>
                             <p class="card-text">
                                 Please review and sign the RoboJackets membership agreement. This document describes our expectations for your behavior in our facility, when traveling, and when representing RoboJackets.
                             </p>
-                            <a href="{{ route('agreement.render') }}">Sign Electronically</a> or <a href="{{ route('agreement.print') }}">Print</a>
                         @else
                             <h4 class="card-title">
                                 Updated Membership Agreement
                             </h4>
                             <p class="card-text">
                                 We have revised our membership agreement. Please review and sign it to maintain your membership.
```

Comment:
I know this wasn't changed in this PR, but I think this wording is awkward.  I like "
```suggestion
                                The RoboJackets membership agreement has changed since the last time you signed it.  Please review and sign the updated version to continue your membership.
```